### ISSUE TITLE
About Scaden test

### ISSUE THREAD

**Issue by solitudetuzi** [KevinMenden/scaden#115]
Hello, kevin.

I'm very sorry to bother you this year. I am a graduate student and a beginner in the field of single cell. Your scaden's thesis is the first one I have studied. I want to learn about bioinformatics from this yours.
I am studying scaden, I have done the first experiment in your thesis.
Use pbmc data (data6k, data8k, donorA, donorC) to be verified with scaden.
First, I used the pbmc.h5ad data you provided for model training,
Use the data6k_500_samples.txt ， data8k_500_samples.txt， donorA_500_samples.txt donorC_500_samples.txt provided by you as the prediction file, and compare the prediction results with other methods.

Use scaden to predict the above data6k, data8k, donorA, donorC.samples, and compare with other methods to get good results.

I want to try other experiments in your article, such as pbmc1 and pmbc2 data, pancreas, mouse Brain, etc.
You provide downloads of these training data on the website, and use the following data for model training.

But in addition to the first pbmc experiment, the rest of the data set experiments, I found only the prediction.txt and labels .txt class files in the data you provided, after model training, can not predict and verify your conclusions.

I would like to ask you to provide me with a prediction document on this data, namely the sample .txt documents, so that can better study your paper.
This year, due to the lack of data, I can no longer go into further study.
I'm really, really sorry to have been interrupting you. I'd love to learn how your cell anti-convolution is done.
Thank you so much.

**Comment by KevinMenden** [comment]
HI @solitudetuzi ,

sorry for the late response - I was on holiday.

The data for these experiments are taken from other publications and I don't want to just upload them somewhere else.
However, I have noted the sources for these datasets in the methods section - so you should be able to download the data yourself. The supplementary material provides further information about how to merge some of the cell types.

Alternatively, you can look for new datasets with ground truth cell fraction information available - maybe new datasets have been made available since the release of Scaden.

Let me know if you have any questions related to the download of the data. If you have a lot of trouble, feel free to contact me again (also via email: kevin.menden@t-online.de)

Cheers,
Kevin
